config/webpack/webpack.config.js

```diff
@@ -75,7 +75,9 @@ const makeImageLoaders = (options = {}) => {
     {
       loader: require.resolve('url-loader'),
       options: {
-        limit: server ? 999999999 : 10000
+        limit: 10000,
+        fallback: require.resolve('file-loader'),
+        emitFile: !server
```

Comment:
This seems dangerous.
Isn't this relying on the the client code to also importing the image?
If for any reason the file is not loaded client side then it would be a returning a url to a file that doesn't exist.

Is there any harm in outputting the file?
If the files client/render are the same then emitting the file will make no difference.
If the files client/render are different then not emitting the file would resulting in a 404.

Reply:
This is what the existing logic was trying to achieve, by setting a limit of `999999999` on the server, effectively inlining every image.

Server code shouldn't be referencing images, only code shared across client and server.

Reply:
Fair enough. We'll need to revisit this if anyone wants to create a static-only page (no client at all), or a static-only image (perhaps some `<noscript>` type scenario), but what you've got should work in the meantime.

Could we add some comment describing why we don't need to emit the file? For any future developers who are debugging this.

Reply:
@jahredhope I'm a little confused here. Aren't our new verticals for invoicing and shopfront 'static pages' in that the static html files are build via sku and the renderToString functionality of react before it's deployed to and served from S3?

If I turn off JS on shopfront so that the client JS doesn't rehydrate I'd still be able to display the images from seekcdn via the static html page.

Anyway, comments have been added :-)